src/element/index.ts

```diff
@@ -41,7 +41,7 @@ export function getSyncableElements(elements: readonly ExcalidrawElement[]) {
 }
 
 export function getElementMap(elements: readonly ExcalidrawElement[]) {
-  return getSyncableElements(elements).reduce(
+  return elements.reduce(
```

Comment:
I removed filtering out elements via `getSyncableElements` from inside the `getElementMap()` helper for 2 reasons:

    1. It's not necessary because we shouldn't be syncing unsyncable elements in the first place (and we're not).
    2. We shouldn't be doing any filtering in a method that's supposed to simply transform an `array` → `object` because that goes against expectations. In fact, this was one of the issues partially causing the arrow bug above.

Reply:
+1